packages/web-react/src/types/link.ts

```diff
@@ -15,6 +16,8 @@ export interface LinkElementTypeProps<T extends ElementType = 'a'> {
 export interface LinkProps<T extends ElementType = 'a'> extends LinkElementTypeProps<T>, ChildrenProps, StyleProps {
   /** Link's href attribute */
   href: string;
+  /** Link's target attribute */
+  target: LinkTarget;
```

Comment:
```suggestion
  target?: LinkTarget;
```

This type should not be required. Run `yarn types` to see that you break type system in a few components.